src/layers/legacy/medCoreLegacy/database/medAbstractDatabaseImporter.cpp

```diff
@@ -303,15 +327,9 @@ void medAbstractDatabaseImporter::importFile ( void )
     // from now on the process cannot be cancelled
     emit disableCancel ( this );
 
-    // 3) Re-read selected files and re-populate them with missing metadata
-    //    then write them to the database, and populate db tables
-    QMap<QString, QStringList>::const_iterator it = imagesGroupedByVolume.begin();
-    QMap<QString, QString>::const_iterator  itPat = imagesGroupedByPatient.begin();
-    QMap<QString, QString>::const_iterator  itSer = imagesGroupedBySeriesId.begin();
-
-    // 3.1) first check is after the filtering we have something to import
-    // maybe we had problems with all the files, or they were already in the database
-    if ( it == imagesGroupedByVolume.end() )
+    // 3) First check is after the filtering we have something to import.
```

Comment:
i do not understand this comment ^^'

Reply:
Me neither. I'm afraid of changing this comment because I'm not sure what they meant.

Reply:
AH, it could be "First, check IF after the filtering we have something to import"